src/Services/Forms/InlineRepeater.php

```diff
@@ -149,6 +150,13 @@ public function max(int $max): static
         return $this;
     }
 
+    public function buttonAsLink(bool $buttonAsLink): static
```

Comment:
```suggestion
    public function buttonAsLink(bool $buttonAsLink = true): static
```

For boolean properties like this we can set a default to true so that one can call the method without having to pass a boolean.